plugins/mcreator-localization/help/default/entity/trading_type.md

```diff
@@ -0,0 +1,4 @@
+This parameter defines the entity's trading type.
+
+* **Villager:** The entity has levels (and therefore, a progress bar), like vanilla villagers. However, there is no profession block.
```

Comment:
Usually no colon is here, but - outside bold instead

![image](https://github.com/MCreator/MCreator/assets/16374228/44f93e6e-98c0-491e-a22f-de5eef591e63)

Reply:
Done!

Reply:
Thanks!